### DISCUSSION TITLE
NVIC on Polarfire SoC

### DISCUSSION THREAD

**Discussion by andrenemat** [polarfire-soc/polarfire-soc-documentation#292] [General] [answered]
Do we have an implementation of an NVIC or something similar on Polarfire SoC? I understand we have the CLINT and the PLIC and have seen some articles online at attempting something like a NVIC on RISC-V processors.

**Comment by TommyMurphyTM1234** [comment, marked as answer]
Perhaps you're thinking of the RISC-V Core-Local Interrupt Controller (CLIC), which is an extension whose specification is still active work in progress and, consequently, still only a draft?

* https://github.com/riscv/riscv-fast-interrupt

**Reply by andrenemat** [reply]
Thanks Tommy, appreciate it.
